### PR TITLE
ProjectJupyter.JupyterLab version 3.5.0-1

### PR DESCRIPTION
**Pull request by brl0**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90646)

### CODE REVIEW

**Comment by brl0** [comment]
@microsoft-github-policy-service agree

File: manifests/p/ProjectJupyter/JupyterLab/3.5.0-1/ProjectJupyter.JupyterLab.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ProjectJupyter.JupyterLab
+PackageVersion: 3.5.0-1
+Installers:
+- Architecture: x86
```
Comment (Trenly) [inline manifests/p/ProjectJupyter/JupyterLab/3.5.0-1/ProjectJupyter.JupyterLab.installer.yaml:7]:
```suggestion
- Architecture: x64
```
image

File: manifests/p/ProjectJupyter/JupyterLab/3.5.0-1/ProjectJupyter.JupyterLab.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ProjectJupyter.JupyterLab
+PackageVersion: 3.5.0-1
+PackageLocale: en-US
+Publisher: Project Jupyter
+PackageName: JupyterLab
+License: BSD-3-Clause license
+Copyright: Copyright © 2022 Project Jupyter
+ShortDescription: JupyterLab Desktop
```
Comment (Trenly) [inline manifests/p/ProjectJupyter/JupyterLab/3.5.0-1/ProjectJupyter.JupyterLab.locale.en-US.yaml:11]: Please provide a better short description of what JupyterLab does

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by brl0** [comment]
Closing to reopen later.

**Pull request closed without merging**